### PR TITLE
sec 5

### PR DESCRIPTION
**Pull request by kavindu0**

### CODE REVIEW

File: client/src/app/components/home/home.component.html [hunk 1]
```diff
@@ -143,6 +143,81 @@ <h2>Create your next Project with us</h2>
   </div>
 </div>
 
+<div class="sec-5">
+  <div class="container">
+    <div class="d-md-flex align-items-end justify-content-between w-100">
```
Comment (kavindu0) [inline client/src/app/components/home/home.component.html:148]: add play cards medium flex and small screens block

**Review by kavindu0 — COMMENTED** [review]
by kavindu0

File: client/src/app/components/home/home.component.html (hunk 1 above)
Comment (kavicastelo) [reply client/src/app/components/home/home.component.html:148]: okay

File: client/src/app/components/home/home.component.html [hunk 2]
```diff
@@ -143,6 +143,81 @@ <h2>Create your next Project with us</h2>
...
+      <div class="div-5-cont-1">
+        <button class="popup-video">
+                              <span class="play-btn" (click)="loadVideo('https://www.youtube.com/embed/yfK5PRth85Y?si=yYI_pyf-Y4OK9xNa')">
```
Comment (kavicastelo) [inline client/src/app/components/home/home.component.html:151]: create loadVideo method and pass url value to it

File: client/src/app/components/home/home.component.html [hunk 3]
```diff
@@ -143,6 +143,81 @@ <h2>Create your next Project with us</h2>
...
+                                 <svg fill="#ffffff" height="200px" width="200px" version="1.1" id="Layer_1"
```
Comment (kavicastelo) [inline client/src/app/components/home/home.component.html:152]: used play button svg

File: client/src/app/components/home/home.component.scss [hunk 4]
```diff
@@ -281,6 +281,68 @@ nav {
   }
 }
 
+.sec-5{
```
Comment (kavicastelo) [inline client/src/app/components/home/home.component.scss:284]: section 5 styles are started from here

File: client/src/app/components/home/home.component.ts [hunk 5]
```diff
@@ -1,5 +1,6 @@
-import {Component, HostListener, OnInit} from '@angular/core';
+import {Component, HostListener, OnInit, ChangeDetectorRef, ElementRef, Renderer2} from '@angular/core';
```
Comment (kavicastelo) [inline client/src/app/components/home/home.component.ts:1]: used following imports to handle event body click

File: client/src/app/components/home/home.component.ts [hunk 6]
```diff
@@ -1,5 +1,6 @@
...
 import VanillaTilt from 'vanilla-tilt';
+import {DomSanitizer, SafeResourceUrl} from "@angular/platform-browser";
```
Comment (kavicastelo) [inline client/src/app/components/home/home.component.ts:3]: used following import statements to handle safe urls

File: client/src/app/components/home/home.component.ts [hunk 7]
```diff
@@ -29,6 +33,11 @@ export class HomeComponent implements OnInit{
     }
   }
 
+  @HostListener('document:click', ['$event'])
```
Comment (kavicastelo) [inline client/src/app/components/home/home.component.ts:36]: event body click handling method

File: client/src/app/components/home/home.component.ts [hunk 8]
```diff
@@ -45,13 +54,26 @@ export class HomeComponent implements OnInit{
 
   vanillaTiltAnimation() {
     const element = document.querySelector('.sec-3-image') as HTMLElement;
+    const element2 = document.querySelectorAll('.div-5-cont-1');
```
Comment (kavicastelo) [inline client/src/app/components/home/home.component.ts:57]: select all `.div-5-cont-1` elements and added vanilla tilts styles to them

File: client/src/app/components/home/home.component.ts [hunk 9]
```diff
@@ -45,13 +54,26 @@ export class HomeComponent implements OnInit{
...
+    // @ts-ignore
+    VanillaTilt.init(element2, {
+      max: 15,
+      speed: 100,
+      glare: false,
+      'max-glare': 0.5
+    })
   }
 
+  loadVideo(video: string) {
```
Comment (kavicastelo) [inline client/src/app/components/home/home.component.ts:75]: load video method

**Review by kavicastelo — COMMENTED** [review]
by kavi

**Comment by kavindu0** [comment]
got them

**Pull request merged**